infra/src/test/kotlin/fi/fta/geoviite/infra/geocoding/GeocodingTest.kt

```diff
@@ -97,20 +99,8 @@ val alignment = alignment(segment1, segment2, segment3)
 val startAddress = TrackMeter(KmNumber(2), 150)
 val kms =
     listOf(
-        GeocodingKm(
```

Comment:
Samat fiksit näihin pariin testiluokkaan kuin tein jo aiemmin maininkin puolelle. En ole ihan varma miksi nämä välillä kääntyi vanhassakin muodossa mutta nyt ainakin minulla failaa kyllä aina ilman näitä muutoksia.